phoebe/atmospheres/passbands.py

```diff
@@ -631,48 +624,45 @@ def load(cls, archive, load_content=True, init_extrapolation=True):
                 for atm in ['ck2004', 'phoenix', 'tmap']:
```

Comment:
I'm wondering if we could do something similar for pb.save? This updated way of loading the passband is much more efficient when it comes to adding new atmospheres.  Can we update save and then make the list of atmospheres a global list that we define elsewhere? Like _supported_atms but without blackbody and extern_*?

Reply:
Yes, that would be awesome! In fact, I want to move away from individual axes and tables and store everything in Ndpolator class instances. TBD.